source/downloads.html

```diff
@@ -147,6 +159,17 @@ <h2 id="prior-releases">Prior Releases</h2>
       <a href="{{ site.wiki_url }}/Version+Notes+2.5.12">Version notes</a>
     </td>
   </tr>
+  <tr>
+    <td class="no-wrap">
+      Struts 2.3.33
+    </td>
+    <td class="no-wrap">July 2017</td>
+    <td>
+    </td>
+    <td>
+      <a href="/docs/version-notes-2333.html">Version notes</a>
```

Comment:
Could you use `{{ site.wiki_url }}/Version+Notes+2.3.33`?

Reply:
All 3 comments are about the same switch to `{{}}` https://github.com/apache/struts-site/pull/63#discussion_r155154531

The style `/docs/version-notes-2332.html` works as of now.

Reply:
It works because there is a redirect defined in .htaccess but I would like to finally get rid of it :)